Stripe/UINavigationBar+Stripe_Theme.m

```diff
@@ -18,6 +19,14 @@ - (void)stp_setTheme:(STPTheme *)theme {
     [self stp_artificialHairlineView].backgroundColor = theme.tertiaryBackgroundColor;
     self.barTintColor = theme.secondaryBackgroundColor;
     self.tintColor = theme.accentColor;
+
+    if ([STPColorUtils colorIsBright:theme.secondaryBackgroundColor]) {
```

Comment:
The documentation is a little confusing, but as far as I can tell:
1. If you are a translucent bar, it only affects status bar rendering (this is our case).
2. If you are not translucent and you have a non-opaque background image, this also controls whether the background color behind that image is white or black. If you're using tintcolor instead of a bg image it appears to do nothing.